.circleci/config.yml

```diff
@@ -69,7 +72,7 @@ commands:
     - run: npm run cov:e2e:report || true
     - run: npm run cov:e2e:<<parameters.suite>>:publish
 orbs:
-  node: circleci/node@4.9.0
+  node: circleci/node@5.1.0
```

Comment:
this new version upgrades to latest available nvm version _before_ attempting an install